docs/node.md

```diff
@@ -62,28 +62,30 @@ After creation, Kubernetes will check whether the node is valid or not.
 For example, if you try to create a node from the following content:
 ```json
 {
-  "id": "10.1.2.3",
-  "kind": "Minion",
-  "apiVersion": "v1beta1",
-  "resources": {
-    "capacity": {
-      "cpu": 1000,
-      "memory": 1073741824
-    },
-  },
-  "labels": {
-    "name": "my-first-k8s-node",
+  "kind": "Node",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "10.240.79.157",
+    "labels": {
+      "name": "my-first-k8s-node"
+    }
   },
+  "status": {
```

Comment:
I am not sure how it was in v1beta1, but the api object has evolved quite a lot after that.

The capacity is maintained by the node (kubelet) itself, and I don't think user should be writing to that at all.